### ISSUE TITLE
Upload on save doesn't work

### ISSUE THREAD

**Issue by dpedu** [Equals182/eqFTP#119]
Brackets: Release 1.2 build 1.2.0-15697 (release 8f82e2a97) 
Version: 00cbfad6ba (Brackets reports 0.7.0)

Nothing related to eqFTP is logged in the developer tools besides: `[eqFTP] Connected to server`. I'm using SFTP mode.

**Comment by Equals182** [comment]
Hi. First of all, please update from here: `https://github.com/Equals182/Buggy-eqFTP` and then restart Brackets manually.

Do you see logs in DevTools when you just connecting to server or navigating through folders? There should be tons of `[eqFTP-ftpDomain]`.

**Comment by dpedu** [comment]
I installed that version and I'm seeing messages logged when browsing:

```
[node-log 10:23:44 AM] [eqFTP-queueisbusy][q.p just started] Setting busy to true
NodeDebugUtils.js:117 [node-log 10:23:44 AM] [eqFTP-queueisbusy][q.p folder] Setting busy to false
NodeDebugUtils.js:117 [node-log 10:23:44 AM] [eqFTP-queueisbusy][q.p just started] Setting busy to true
NodeDebugUtils.js:117 [node-log 10:23:44 AM] [eqFTP-queueisbusy][q.p folder] Setting busy to false
NodeDebugUtils.js:117 [node-log 10:23:47 AM] [eqFTP-queueisbusy][s.sKA 1] Setting busy to true
NodeDebugUtils.js:117 [node-log 10:23:47 AM] Keep Alive (cd .) SFTP
NodeDebugUtils.js:117 [node-log 10:23:47 AM] [eqFTP-queueisbusy][s.sKA 2] Setting busy to false
```

When I save a file (externally or via brackets), nothing is logged. If I right click a file and add it to queue manually, it uploads.

While testing the above I also noticed this - symbolic links are completely absent in eqftp's file listing.

**Comment by Equals182** [comment]
No symlink support yet.
I'll make some extra logs for you. Can you post a screenshot of **Automatization** tab in Connection settings?

**Comment by dpedu** [comment]
Here you go: http://i.imgur.com/lNwW5w4.png

I've tried a few different combinations of these settings.

**Comment by Equals182** [comment]
With **Add to paused queue** on, your files should be added to queue on save instead of direct upload. 

Please update from here: `https://github.com/Equals182/Buggy-eqFTP`. You should be able to see `[eqFTP-uploadonsave]` logs. Please post them here from very first to the last.

**Comment by dpedu** [comment]
Hey,

I re-installed eqFTP using that github url, relaunched Brackets then: Browsed the remote server then opened a few local files and saved them. Nothing is queued with that setting on or off. *Everything* logged is: 

```
Download the React DevTools for a better development experience: http://fb.me/react-devtools
/utils/DeprecationWarning.js:90 Deprecated: Do not use $().on/off() on Brackets modules and model objects. Call on()/off() directly on the object without a $() wrapper.
    at new jQuery.fn.init (/brackets.js:481:36)
    at n (file:///Applications/Brackets.app/Contents/www/thirdparty/thirdparty.min.js:559:405)
    at file:///Users/dave/Library/Application%20Support/Brackets/extensions/user/renanveroneze.brackets-open-terminal-here/main.js:42:9
    at _callHandler (/utils/AppInit.js:93:13)
    at _addListener (/utils/AppInit.js:128:13)
    at Object.htmlReady (/utils/AppInit.js:150:9)
    at Object.<anonymous> (file:///Users/dave/Library/Application%20Support/Brackets/extensions/user/renanveroneze.brackets-open-terminal-here/main.js:33:13)
/utils/DeprecationWarning.js:90 Deprecated: Do not use $().on/off() on Brackets modules and model objects. Call on()/off() directly on the object without a $() wrapper.
    at new jQuery.fn.init (/brackets.js:481:36)
    at n (file:///Applications/Brackets.app/Contents/www/thirdparty/thirdparty.min.js:559:405)
    at new ProjectObserver (file:///Users/dave/Library/Application%20Support/Brackets/extensions/user/dnbard.projects/services/projectObserver.js:7:20)
    at Object.<anonymous> (file:///Users/dave/Library/Application%20Support/Brackets/extensions/user/dnbard.projects/main.js:7:27)
/LiveDevelopment/MultiBrowserImpl/transports/NodeSocketTransport.js:87 NodeSocketTransport - start
/utils/DeprecationWarning.js:90 Deprecated: Do not use $().on/off() on Brackets modules and model objects. Call on()/off() directly on the object without a $() wrapper.
    at new jQuery.fn.init (/brackets.js:481:36)
    at n (file:///Applications/Brackets.app/Contents/www/thirdparty/thirdparty.min.js:559:405)
    at file:///Users/dave/Library/Application%20Support/Brackets/extensions/user/equals182.eqftp/main.js:1622:9
    at _callHandler (/utils/AppInit.js:93:13)
    at Object._dispatchReady (/utils/AppInit.js:113:13)
    at Object.eval (/brackets.js:291:33)
    at j (file:///Applications/Brackets.app/Contents/www/thirdparty/thirdparty.min.js:559:26911)
    at Object.k.add [as done] (file:///Applications/Brackets.app/Contents/www/thirdparty/thirdparty.min.js:559:27220)
    at Object.d.always (file:///Applications/Brackets.app/Contents/www/thirdparty/thirdparty.min.js:559:28096)
extensions/default/DebugCommands/NodeDebugUtils.js:117 [node-log 10:03:30 AM] [eqFTP-ftpDomain] Loaded
/Users/dave/Library/Application%20Support/Brackets/extensions/user/equals182.eqftp/main.js:1613 [eqFTP] Loaded (v0.7.0)
NodeDebugUtils.js:117 [node-log 10:06:24 AM] [eqFTP-queueisbusy][q.p just started] Setting busy to true
main.js:1891 [eqFTP] Connected to server
NodeDebugUtils.js:117 [node-log 10:06:25 AM] [eqFTP-queueisbusy][q.p folder] Setting busy to false
NodeDebugUtils.js:117 [node-log 10:06:35 AM] [eqFTP-queueisbusy][s.sKA 1] Setting busy to true
NodeDebugUtils.js:117 [node-log 10:06:35 AM] Keep Alive (cd .) SFTP
NodeDebugUtils.js:117 [node-log 10:06:35 AM] [eqFTP-queueisbusy][s.sKA 2] Setting busy to false
```

**Comment by Equals182** [comment]
`[eqFTP-uploadonsave]` should appear only when you save file. It's weird that almost *nothing* appear in logs though.

**Comment by Equals182** [comment]
### Hello there!

I know this issue been there for a pretty long time, but I have some good news here!
We just released eqFTP v**1.0.0-beta.0** (v1-b for short I guess) and we're completely discontinuing support of all previous versions of this plugin.
New version works great on Brackets 1.8 and it should be good on lower versions too.

So yeah, we're closing and locking this issue forever and looking forward for you to try out our new version of eqFTP.
I'm sorry if I somehow disturbed you, or you lost your interest in eqFTP or Brackets in general, we're not going to make any noise here again.

If you have any questions, please contact me via email (you can find it in [my profile](https://github.com/Equals182)) or [Telegram app](http://t.me/equals182)

Thank you for understanding 😊
